Arelle/EdgarRenderer
# update TaxonomyAddonManager and ix viewer

#### Reason for change
EdgarRenderer for 23.1 TaxonomyAddonManager was missing the 2023 taxonomies doc and ref linkbase files.  Also EDGAR team updated ix viewer.

#### Description of change
Add 2023 doc and ref linkbases to resources/TaxonomyAddonManager.py

#### Steps to Test
Verify 2023 doc and ref's are showing for an SEC or FASB concept in R file right-mouse button popup and ix viewer fact popup dialog.

Verify that EDGAR team's latest ix viewer change behaves as expected.

**review**:
@Arelle/arelle
